### ISSUE TITLE
add a X button to clear the current image from clipboard

### ISSUE THREAD

**Issue by radonmiser** [FreeLanguageTools/vocabsieve#165]
**Is your feature request related to a problem? Please describe.**
It takes multiple step to clear the image in the UI if it's unwanted. User need to either
1.restart the program
2.deleted the unwanted image after it's in the anki note.

**Describe the solution you'd like**
A button to clear the current image.

**Additional context**

**Comment by Mycheze** [comment]
I agree there should be an X button, but for now, you can use the shortcut `Ctrl + W`.
